assets/js/common/OperationModals/SaptuneSolutionApplyModal.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from 'react';
+import { filter } from 'lodash';
+import Select from '@common/Select';
+import OperationModal from './OperationModal';
+
+const NOT_SELECTED = 'Select a saptune solution';
+
+const solutions = [
+  {
+    value: NOT_SELECTED,
+    key: NOT_SELECTED,
```

Comment:
NOT_SELECTED contains spaces in the string, which is allowed, but i was wondering if we should not have something without spaces? 
``` key: "not_selected" ```
Tbh it is more a question xD

Reply:
Changed. It looks better indeed